addon/templates/components/cfb-form-editor/question/options.hbs

```diff
@@ -1,15 +1,29 @@
 {{component labelComponent}}
 
-<ul class="uk-list uk-list-divider uk-margin-remove-bottom uk-margin-small-top">
-  {{#each optionRows as |row i|}}
+{{#uk-sortable
+  threshold=".uk-sortable-handle"
```

Comment:
I'm 100% sure this won't work - did you test it??

Reply:
Yes tested it and it works

Reply:
Pretty sure what you're passing there is supposed to be the `handle`. `threshold` is a number - there's a typo in the documentation that might've caused this error here but this certainly doesn't work.

Reply:
Changed it to handle